test/ggsn_proxy_SUITE.erl

```diff
@@ -620,8 +621,8 @@ common() ->
      session_accounting,
      sx_upf_reconnect,
      sx_upf_removal,
-     sx_timeout,
-     delete_bearer_requests_multi
+     sx_timeout
+     %% delete_bearer_requests_multi
```

Comment:
Why test case `delete_bearer_requests_multi` is disabled?

Reply:
that change is actually in PR #257 (which need to be merge first), Reason is explained in the test case.